documentation/source/user_section/tutorials/2/2-9.rst

```diff
@@ -0,0 +1,76 @@
+.. _customizing-registration-section:
+
+2.9 Customizing the registration command
+########################################
+
+This page provides recommendations for how to adjust the registration commands if the default parameters are insufficient for your specific data and pipeline.
+
+The ``-param`` argument
+***********************
+
+The ``-param`` argument defines the transformations for each step of the registration process. The easiest way to use ``-param`` is to start with the default values, and adjust one parameter at a time.
+
+.. code-block::
+
+   # Default values for '-param'. (Note: Normally, this would be written as one line, but a line break was added for readability.)
+   -param step=1,type=imseg,algo=centermassrot,metric=MeanSquares,iter=10,smooth=0,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6:
+          step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,smooth=1,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6
```

Comment:
Does this all have to be run together as a single long string? Can you use `\` to split it up?

```suggestion
   # Default values for '-param'. (Note: Normally, this would be written as one line, but a line break was added for readability.)
   -param step=1,type=imseg,algo=centermassrot,metric=MeanSquares,iter=10,smooth=0,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6 \
step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,smooth=1,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6
```

Reply:
I think it's possible. But, for this specific snippet, it's not actually a command to be run. 

Because it's just showing the argument for display purposes, I wanted to preserve the indentation so it would be displayed nicely. And, as your suggestion shows, [`\` can't really do this](https://stackoverflow.com/questions/23929235/multi-line-string-with-extra-space-preserved-indentation). :sweat: